src/zenml/integrations/azure/secrets_managers/azure_secrets_manager.py

```diff
@@ -199,18 +142,24 @@ def update_secret(self, secret: BaseSecretSchema) -> None:
         """
         self._ensure_client_connected(self.key_vault_name)
 
-        adjusted_content = prepend_group_name_to_keys(secret)
+        for key, value in secret.content.items():
+            encoded_name = base64.b64encode(
```

Comment:
`encoded_name` feels like maybe it's the wrong name for it? aren't we talking about what the key is called here? so maybe call it `encoded_key` or `encoded_key_name`?

Reply:
Good point, I updated it